src/components/NyPlanSide/FyllUtPlanSteg/UtkastLagringInfo.tsx

```diff
@@ -1,50 +1,62 @@
 import { BodyLong, HStack, HelpText, Loader } from "@navikt/ds-react";
+import {
+  getFormattedDateAndTimeString,
+  getFormattedTimeString,
+} from "@/ui-helpers/dateAndTime";
+import {
+  isDateToday,
+  isNotMoreThanOneHourAgo,
+} from "@/utils/dateAndTime/dateUtils";
 
 interface Props {
   isSavingUtkast: boolean;
-  utkastSistLagretTidspunkt: Date | null;
+  utkastSistLagretTidspunkt: string | null;
 }
 
 export default function UtkastLagringInfo({
   isSavingUtkast,
   utkastSistLagretTidspunkt,
 }: Props) {
-  if (!isSavingUtkast && !utkastSistLagretTidspunkt) {
-    return null;
+  if (isSavingUtkast) {
+    return (
+      <HStack gap="4" align="center">
+        <BodyLong size="medium" className="text-ax-text-neutral-subtle">
+          <span>Lagrer utkast...</span>
+        </BodyLong>
+
+        <Loader
+          size="small"
+          title="Lagrer utkast"
+          variant="interaction"
+          className="relative -top-px"
+        />
+      </HStack>
+    );
+  }
+
+  if (utkastSistLagretTidspunkt) {
+    const isToday = isDateToday(utkastSistLagretTidspunkt);
+    const displaySeconds = isNotMoreThanOneHourAgo(utkastSistLagretTidspunkt);
+
+    const sistLagretTidspunktFormatted = isToday
+      ? `kl. ${getFormattedTimeString(utkastSistLagretTidspunkt, displaySeconds)}`
+      : getFormattedDateAndTimeString(utkastSistLagretTidspunkt);
+
+    return (
+      <HStack gap="4" align="center">
+        <BodyLong size="medium" className="text-ax-text-neutral-subtle">
+          <span>Utkast sist lagret {sistLagretTidspunktFormatted}</span>
+        </BodyLong>
+
+        <HelpText className="relative -top-px">
+          Dine endringer lagres som et utkast mens du skriver. Du finner igjen
+          utkastet på siden med oppfølgingsplaner for den ansatte. Klikk på
+          «Avslutt og fortsett senere» for å avslutte for nå og gå til den
+          siden. Derfra kan du klikke på utkastet for å fortsette skrivingen.
```

Comment:
Det er vel ikke endret på i denne PRen, men teksten her var kanskje litt rar med " Klikk på
          «Avslutt og fortsett senere» for å avslutte for nå og gå til den siden." - Hva betyr "den siden"? 

og "for å fortsette skrivingen" var litt muntlig kanskje. Tenker kanskje det holder å ha noe ala "Dine endringer lagres fortløpende mens du skriver. Dersom du har behov for å jobbe videre med oppfølgingsplanen senere kan du velge "Avslutt og fortsett senere"."